data-in-pipeline/infra/__main__.py

```diff
@@ -117,13 +173,38 @@
         publicly_accessible=False,
         auto_minor_version_upgrade=True,
         tags=tags,
+        opts=pulumi.ResourceOptions(
+            depends_on=[aurora_cluster],
+        ),
     )
-    for i in range(2)
+    for i in range(max_instances)
 ]
 
+pulumi.export(
+    f"{name}-{environment}-aurora-cluster-name", aurora_cluster.cluster_identifier
+)
+pulumi.export(f"{name}-{environment}-aurora-cluster-arn", aurora_cluster.arn)
+pulumi.export(
+    f"{name}-{environment}-aurora-cluster-resource-id",
+    aurora_cluster.cluster_resource_id,
+)
+pulumi.export(
+    f"{name}-{environment}-aurora-instance-ids",
+    [instance.identifier for instance in aurora_instances],
+)
+pulumi.export(f"{name}-{environment}-aurora-endpoint", aurora_cluster.endpoint)
+pulumi.export(
+    f"{name}-{environment}-aurora-reader-endpoint", aurora_cluster.reader_endpoint
+)
+
```

Comment:
☝️ @katybaulch are we planning on using these anywhere? I do not think we will as most of these values will be used in this stack.

If not can we remove them as we might create inadvertent (over explicit) couplings. 

I have always seen exports as the equivalent to API contracts. We do not currently have a way of letting know any upstream services these values have changed, so if they do couple to them we can potentially break those services, so it might be best to avoid them till them.

We have seen this with the bastion updating which has broken other services in our stack.